### PR TITLE
Install Instructions for Beginners

### PR DESCRIPTION
**Pull request by thetayloredman**
Original: #34

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,21 +9,125 @@
 ## About
 FurDevs Discord Bot is a Multipurpose Discord Bot  for the [FurDevs Discord Server](https://discord.gg/JABY6Htnqt)
 
-## Installation
-To Install FurDevs Discord Bot in your system
+## Quick Start
+This section is for you if you're already familiar with discord bots in general.
```
Comment (thetayloredman) [inline README.md:13]:
```suggestion
## Quick Start

This section is for you if you're already familiar with discord bots in general.
```

Markdown items should have leading+trailing newlines like this

File: README.md [hunk 2]
```diff
@@ -9,21 +9,125 @@
...
+
+To use the FurDevs Discord Bot on your server, you'll need:
 
 - Git, NPM or Yarn, and Node
```
Comment (thetayloredman) [inline README.md:17]:
```suggestion
- Git, NPM or Yarn, and Node.js
```

File: README.md [hunk 3]
```diff
@@ -9,21 +9,125 @@
...
 - You Must Have a Discord Account
```
Comment (thetayloredman) [inline README.md:18]:
```suggestion
- A Discord account
```

File: README.md [hunk 4]
```diff
@@ -9,21 +9,125 @@
...
 - MongoDB Account, and a Database for FurDevs
```
Comment (thetayloredman) [inline README.md:19]:
```suggestion
- MongoDB Account, and a database for FurDevs
```

File: README.md [hunk 5]
```diff
@@ -9,21 +9,125 @@
...
 - An Application for the Bot (This can be made in the [Discord Developer Portal](https://discord.com/developers/applications))
 - Common Sense ( Don't leak the bots token )
```
Comment (thetayloredman) [inline README.md:21]:
```suggestion
- An application for the bot (Make this at the [Discord Developer Portal](https://discord.com/developers/applications))
- Common Sense (Don't leak the bot token)
```

File: README.md [hunk 6]
```diff
@@ -9,21 +9,125 @@
...
 
+Once you have all the dependencies:
+
+1. `git clone git@github.com:Discord-FurDevs/FurDevs-Bot.git`
```
Comment (thetayloredman) [inline README.md:25]:
```suggestion
1. `git clone https://github.com/Discord-FurDevs/FurDevs-Bot.git`
```

File: README.md [hunk 7]
```diff
@@ -9,21 +9,125 @@
...
+2. Going to that directory then do `npm i` to install all of its dependicies.
```
Comment (thetayloredman) [inline README.md:26]:
```suggestion
2. Run `npm i`
```

File: README.md [hunk 8]
```diff
@@ -9,21 +9,125 @@
...
+3. You'll find a `.env.example`, Rename it to `.env` then add all of the neccessary information.
+4. Start the bot by doing `node .` or `node index.js`.
+
+# Verbose Installation Guide
```
Comment (thetayloredman) [inline README.md:30]:
```suggestion
## Verbose Installation Guide

```

Top level headers should only appear once on a document.

File: README.md [hunk 9]
```diff
@@ -9,21 +9,125 @@
...
+If you're not terribly familiar with discord bot development, then this section may be for you. It has a little more detail than the above section, and focuses on getting things working in Windows 10.
+
+Of course, if you'd like help, head on over to the FurDevs discord server. :)
```
Comment (thetayloredman) [inline README.md:33]:
```suggestion
Of course, if you'd like help, head on over to the FurDevs Discord server. :)
```

Discord is a proper noun

File: README.md [hunk 10]
```diff
@@ -9,21 +9,125 @@
...
+
+## Overview
```
Comment (thetayloredman) [inline README.md:35]:
```suggestion
### Overview

```

File: README.md [hunk 11]
```diff
@@ -9,21 +9,125 @@
...
+Discord bots are programs that run on internet connected computers. They talk to the Discord API to do things like send messages in a Discord server.
+
+To add the FurDevs Discord Bot to a server, you'll have to:
+
+1. Have a Discord account.
+2. Have a Discord server you want to add it to. (You have to have the proper permissions to do this. If you're just testing out the bot, then you should probably make an empty one for testing purposes.)
+3. Have a copy of the FurDevs Discord Bot running on your computer.
+
+## FurDevs Discord Bot Prerequisites
```
Comment (thetayloredman) [inline README.md:44]:
```suggestion
### FurDevs Discord Bot Prerequisites

```

File: README.md [hunk 12]
```diff
@@ -9,21 +9,125 @@
...
+To run the FurDevs Discord Bot on your computer, you'll need to install the following if they're not on your Windows 10 system already. Some have lots of options in the installation process - just accept all the defaults and click next until the installation is complete and you should be fine.
+
+- [Git](https://git-scm.com/download/win)
+- [Node.js](https://nodejs.org/en/download/current/)
+- [MongoDB Community](https://www.mongodb.com/try/download/community)
+
```
Comment (thetayloredman) [inline README.md:50]:
```suggestion
```

File: README.md [hunk 13]
```diff
@@ -9,21 +9,125 @@
...
+
+### MongoDB Community Installation Notes
```
Comment (thetayloredman) [inline README.md:52]:
```suggestion
#### MongoDB Community Installation Notes

```

File: README.md [hunk 14]
```diff
@@ -9,21 +9,125 @@
...
+When you install MongoDB Community with the defaults, a MongoDB service is installed on your Windows 10 installation. When you're done using mongodb for this bot (or other purposes) it makes sense to disable it. To do that, type "services.msc" in the start menu, press enter, scroll down to and click "MongoDB Server", click the "Stop" button, then change the "Startup type" to "Disabled".
```
Comment (thetayloredman) [inline README.md:53]:
```suggestion
When you install MongoDB Community with the defaults, a MongoDB service is installed on your Windows 10 installation. When you're done using MongoDB for this bot (or other purposes) it makes sense to disable it. To do that, type "services.msc" in the start menu, press enter, scroll down to and click "MongoDB Server", click the "Stop" button, then change the "Startup type" to "Disabled".
```

File: README.md [hunk 15]
```diff
@@ -9,21 +9,125 @@
...
+
+You'll also install MongoDB Compass. This is just a tool for looking at what's stored in your instance of MongoDB. You don't have to use this to run the FurDevs Discord Bot, but it's nice to have around in case you want to see what's stored in the database. All you have to do to use it is connect to the database by putting the following in the connect box and the pressing the connect button:
+`mongodb://127.0.0.1:27017`
+
+## Getting a copy of the bot
```
Comment (thetayloredman) [inline README.md:58]:
```suggestion
### Getting a copy of the bot

```

File: README.md [hunk 16]
```diff
@@ -9,21 +9,125 @@
...
+Now that you have the prerequisites installed, let's get a copy of the bot.
+
+In a folder that you'd like the bot to be stored in, right click and select the "Git Bash Here" option. Then run the following: (leave it open, we'll use it for subsequent steps)
+`git clone git@github.com:Discord-FurDevs/FurDevs-Bot.git`
```
Comment (thetayloredman) [inline README.md:62]:
```suggestion
`git clone https://github.com/Discord-FurDevs/FurDevs-Bot.git`
```

File: README.md [hunk 17]
```diff
@@ -9,21 +9,125 @@
...
+
+You should see that a new folder was created falled "FurDevs-Bot".
+
+## Configuring the bot
```
Comment (thetayloredman) [inline README.md:66]:
```suggestion
### Configuring the bot

```

File: README.md [hunk 18]
```diff
@@ -9,21 +9,125 @@
...
+Before we can run the bot, we have to configure it.
+
+The first part of that is installing the bot's javascript dependencies. In the same "Git Bash" window you left open from before, run the following:
```
Comment (thetayloredman) [inline README.md:69]:
```suggestion
The first part of that is installing the bot's JavaScript dependencies. In the same "Git Bash" window you left open from before, run the following:
```

File: README.md [hunk 19]
```diff
@@ -9,21 +9,125 @@
...
+
+There's an example file that includes the above that you can fill out to be your `.env` file. Just run:
+
+`cp .env.example .env`
+
+Then open the `.env` file in notepad with:
+
+`notepad .env`
+
+### TOKEN
```
Comment (thetayloredman) [inline README.md:92]:
```suggestion
#### TOKEN

```

File: README.md [hunk 20]
```diff
@@ -9,21 +9,125 @@
...
+2. Check the "Bot" box in the middle column of boxes to check.
+3.  In the checkboxes that appear below, check the "Administrator" box under the "General Permissions" column. (Warning: This is fine for a test server, but you should read about Discord Bot permissions before checking the Administrator box for a bot that has actual users.)
+4. Copy the URL generated under the first group of checkboxes.
+5. Paste the URL into your browser.
+6. Select the server you want to add the bot to.
+7. Click "Continue" then "Authorize".
+
+You're done configuring the token!
+
+### DB
```
Comment (thetayloredman) [inline README.md:116]:
```suggestion
#### DB

```

File: README.md [hunk 21]
```diff
@@ -9,21 +9,125 @@
...
+The DB tells the bot where it can find a MongoDB database that it can use to store its data.
+
+This one is easy to configure - you just have to tell the bot to use your local instance of MongoDB by using the following value:
+
+`mongodb://127.0.0.1:27017`
+
+### DICTIONARYAPI
```
Comment (thetayloredman) [inline README.md:123]:
```suggestion
#### DICTIONARYAPI

```

**Review by thetayloredman — COMMENTED** [review]
AAAAAA

**Comment by EchoLynx** [comment]
> Original: #34

Thank you for the improvements!

**Comment by DeveloperDrago** [comment]
Alrighty

**Comment by EchoLynx** [comment]
Well I can't reopen #34, so I guess I'll make #37... *facepalm*

**Pull request closed without merging**